src/status_im2/contexts/quo_preview/profile/profile_card/view.cljs

```diff
@@ -0,0 +1,108 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.view
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.core :as quo2]
+            [quo2.foundations.colors :as colors]
+            [status-im2.contexts.quo-preview.profile.profile-card.style :as style]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]))
+
+(defn profile-card ;; TODO(@mattgle): Add description for a better component understanding
+  [{:keys [show-sign-profile?
+           key-card?
+           profile-picture?
+           name
+           hash
+           emoji-hash
+           on-press-dots?
+           on-press-sign?] :or {show-sign-profile? false
+                                key-card? false}}]
+  [rn/view {:style style/card-container}
+   [rn/view {:style style/card-header}
+    [user-avatar/user-avatar
+     {:full-name         name
+      :profile-picture   profile-picture?
+      :size              :medium
+      :status-indicator? false
+      :ring?             true}] ;; TODO(@mattgle): The ring? is not showing up for some reason
```

Comment:
Can you share more details about the problem? What have you tried so far?

Reply:
Fixed! I added a 48x48 folder in icons and put there the missing image